wgengine/netstack/netstack_linux.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package netstack
+
+import (
+	"os/exec"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
```

Comment:
there is some weird behavior on DSM6 which led to having to special case DSM7
https://github.com/tailscale/tailscale/blob/main/wgengine/router/router_linux.go#L180-L189

Reply:
https://github.com/tailscale/tailscale/issues/2922#issuecomment-946026420

Reply:
Done; PTAL

Reply:
Confirmed that my tailscaled runs as uid 0 on DSM6 and non-0 on DSM7